vllm/config.py

```diff
@@ -665,9 +683,10 @@ def __init__(
         self._verify_args()
 
     def _verify_args(self) -> None:
-        if self.pipeline_parallel_size > 1:
-            raise NotImplementedError(
-                "Pipeline parallelism is not supported yet.")
+        if (self.pipeline_parallel_size > 1
+                and self.distributed_executor_backend == "mp"):
+            raise NotImplementedError("Pipeline parallelism is not supported "
+                                      "yet with multiprocessing.")
```

Comment:
why is it the case? I think `mp` backend can support it as long as the world size is smaller than the number of gpus.

Reply:
Yes, it can eventually be supported, just not intended to be added in this PR. It can be a separate small change to `multiproc_gpu_executor.py`. The MP backend got added after this PR was made.